src/com/xilinx/rapidwright/edif/EDIFPortInstList.java

```diff
@@ -122,13 +122,6 @@ public void _parseAdd(EDIFPortInst e) {
     }
 
     public void _reSortList() {
```

Comment:
Since this is a public API callable by anyone, does it need to/should it have an underscore prefix?

Reply:
Underscores removed